internal/dao/stubs/context_getters.go

```diff
@@ -0,0 +1,45 @@
+package stubs
+
+import (
+	"context"
+	"errors"
+
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+)
+
+type daoStubCtxKeyType int
+
+const (
+	accountCtxKey daoStubCtxKeyType = iota
+	pubkeyCtxKey  daoStubCtxKeyType = iota
+)
+
+var contextReadError = errors.New("the message")
```

Comment:
Export this, perhaps move it to `errors.go` too or just rename this file to `common.go` or whatever and put all errors here.

```suggestion
var ContextReadError = errors.New("DAO stub not found in context, use WithXyzDao() to create one")
```

Reply:
moved to `stub_errors`